Screenbox/Strings/en-US/Resources.resw

```diff
@@ -893,19 +893,22 @@
   <data name="SettingsEnqueueAllFilesInFolderHeader" xml:space="preserve">
     <value>Add all files in folder to queue</value>
   </data>
-  <data name="Auto" xml:space="preserve">
-    <value>Auto</value>
+  <data name="ThemeAuto" xml:space="preserve">
+    <value>Use system setting</value>
   </data>
-  <data name="Dark" xml:space="preserve">
+  <data name="ThemeDark" xml:space="preserve">
     <value>Dark</value>
   </data>
-  <data name="Light" xml:space="preserve">
+  <data name="ThemeLight" xml:space="preserve">
     <value>Light</value>
   </data>
   <data name="SettingsThemeSelectionDescription" xml:space="preserve">
-    <value>Select a theme to match your Windows settings or choose a custom one</value>
+    <value>Change the app's overall color scheme</value>
```

Comment:
I would use the string used by the inbox apps. I would at least remove the mention of "color".

```suggestion
    <value>Select which app theme to display</value>
```

Reply:
I don't think we should have "app theme" to be repeated. The description becomes redundant then.